lectures/lecture-02/slides.md

```diff
@@ -0,0 +1,3 @@
+# Week 1 Slides
+
+{{ 'https://slides.com/arasevic/cmsc389l-week-02' | iframely }}
```

Comment:
Looks like the first slide still mentions CloudFront!

Reply:
fixed